stubs/pluggy/pluggy/_tracing.pyi

```diff
@@ -0,0 +1,20 @@
+from collections.abc import Callable
+from typing import Any
+from typing_extensions import TypeAlias
+
+_Writer: TypeAlias = Callable[[str], None]
```

Comment:
The upstream package has a bad type annotation here. Since the value returned from `TagTracer._writer` is always discarded, this should be

```suggestion
_Writer: TypeAlias = Callable[[str], object]
```

That will get rid of a couple of the mypy_primer errors